package-build.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config/package-bundle/config
+              - config/release-config.yml
+          - kbld:
+              paths:
+              - '-'
+              - .imgpkg/images.yml
+      export:
+      - imgpkgBundle:
+          image: ghcr.io/vmware-tanzu/carvel-secretgen-controller
```

Comment:
So I think this is where my final question is.

Today we publish two things:
1. An image for secretgen-controller
2. A bundle with all the  config to deploy secretgen-controller as a Package/PackageInstall

We therefore have two packages: https://github.com/orgs/vmware-tanzu/packages?repo_name=carvel-secretgen-controller 

In the previous world, we had the value [here](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/2601d33519a61278c258a3664f4f39f49bcfe750/config/release/values-schema.yml#L8) for the image location, and I believe as part of the release this was published. Is this behaviour maintained? I am just seeing the `bundle` package being published here, I am just unsure of whether the image is still being published or not now? I think some users rely on just the image without the bundle aspect. So I'd like to maintain those separately. Right now it looks like we are changing `ghcr.io/vmware-tanzu/carvel-secretgen-controller` to be a bundle whereas right now it's an image.

Hopefully this is clear.